packages/form/src/SelectRow.js

```diff
@@ -39,25 +40,71 @@ export const Popover = renderToLayer(
     )
 );
 
+/**
+ * Generate a value-label map from all `<SelectOption>`s.
+ *
+ * @param {array} fromOptions
+ * @return {Map}
+ */
+function getValueLabelMap(fromChildren = []) {
+    const resultMap = new Map();
+    const options = parseSelectOptions(fromChildren);
+
+    options.forEach(
+        option => resultMap.set(option.value, option.label)
+    );
+    return resultMap;
+}
+
 class SelectRow extends PureComponent {
     static propTypes = {
         label: PropTypes.node.isRequired,
+        asideAll: PropTypes.string,
+        asideNone: PropTypes.string,
+        asideSeparator: PropTypes.string,
         disabled: PropTypes.bool,
+        // <SelectList> props
+        values: SelectList.propTypes.values,
+        defaultValues: SelectList.propTypes.defaultValues,
+        onChange: PropTypes.func,
         // from formRow()
         ineditable: PropTypes.bool,
         rowProps: rowPropTypes,
     };
 
     static defaultProps = {
+        asideAll: 'All',
+        asideNone: '(Unset)',
+        asideSeparator: ', ',
         disabled: false,
+        // <SelectList> props
+        values: SelectList.defaultProps.values,
+        defaultValues: SelectList.defaultProps.defaultValues,
+        onChange: () => {},
         ineditable: false,
```

Comment:
Add `// from formRow()` to separate bellowing props are came from `formRow()`?

Reply:
of course